examples/nextjs/pages/index.tsx

```diff
@@ -22,6 +32,25 @@ const Home: NextPage = () => {
           }}
         />
       </div>
+      <div
+        style={{
+          width: '450px',
+          height: '820px',
+        }}
+      >
+        <NoSSR>
+          <swap-widget
+            colors={JSON.stringify({
+              primary: '#FF4FFF',
+            })}
+            defaultRoute={JSON.stringify({
+              srcChainID: 'osmosis-1',
+              srcAssetDenom:
+                'ibc/1480b8fd20ad5fcae81ea87584d269547dd4d436843c1d20f15e00eb64743ef4',
+            })}>
+          </swap-widget>
+        </NoSSR>
```

Comment:
nextjs does not play nicely with web-components, so i had to disable it to render without errors

Reply:
The other thing that's annoying is the way these attributes are passed, I think I can add a wrapper component and convert these props to individual strings to make it nicer to use

Reply:
Checkout https://npm.im/@lit/react

[`lit`](https://lit.dev/) is a fantastic web components framework

[`@lit/react`](https://npm.im/@lit/react) addresses directly the annoyance of passing attributes you mention.

Reply:
> Checkout https://npm.im/@lit/react
> 
> [`lit`](https://lit.dev/) is a fantastic web components framework
> 
> [`@lit/react`](https://npm.im/@lit/react) addresses directly the annoyance of passing attributes you mention.

Hey @o-az 👋 I took a look at [lit](https://lit.dev/) and [@lit/react](https://www.npmjs.com/package/@lit/react) and from what I understand from reading through the docs is basically is for creating web-components from scratch, and @lit/react is for **consuming/using** them within react.


The lib i'm currently using, [@r2wc](https://www.npmjs.com/package/@r2wc/react-to-web-component) helps to actually take an existing react-component and turn it into a web-component, which I don't believe lit actually does